css/index.css

```diff
@@ -137,10 +270,219 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+
+@media (max-width: 768px) {
+    body, html{
+        width: 100%;
+        margin: 0 0;
+    }
+    .lambda {
+        align-self: center;
+        margin: 30px 0 0 0 ;
+    }
+    
+    header {
+        flex-direction: column;
+        margin: 0 0 0 0;
+    }
+    header nav {
+        width:100%;
+        margin: 30px 0 0 0;
+    }
+
+    .middle-content .boxes .box1 {
```

Comment:
Another mistake with too much repetition. When using media queries You do not need to repeat all those rules all the time. You should only specify what You want to change and rest of rules will remain unchanged and looked up from the top of the file.

Example:

```
.middle-content .boxes .box {
    width: 12.5%;
    height: 10em;
    background: black;
}

@media (max-width: 400px) {
.middle-content .boxes .box {
    width: 50%;
}
}
```

In this example box will still have height 10em and background black when it reaches resolution below 400px, but it's width will change. That way You get rid of repetition and keep Your code DRY.